docs/detections/session-view.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[session-view]]
+== Session view
+This page describes how to enable and use Session View. Session View is an investigation tool that allows you to view and analyze Linux process data organized
+in a tree according to the Linux logical event model, with processes organized by parentage and time of execution.
+It displays events in a highly readable format inspired by the terminal, making it a powerful tool for monitoring
+and investigating remote session activity on your Linux infrastructure, and understanding user and service behaviour.
+It is currently in beta.
+
+[float]
+[[session-view-data]]
+====== Session View displays:
+1. *Interactive and non-interactive processes:* processes and services with or without a controlling tty.
+2. *User information:* the user that executed each session or process, and any exec user changes.
+3. *User Input:* commands entered by a user into the terminal.
+4. *Process and event telemetry:* process information included in the Linux logical event model.
+5. *Nested sessions:* sessions started by processes descended from the entry session.
```

Comment:
I would add Alerts to this list. One of the major use cases for session view is to highlight processes that resulted in security alerts.